### PR TITLE
project scrum updates

### PR DESCRIPTION
**Pull request by iammiracle**

### CODE REVIEW

**Review by fredrikarvas-vertel — APPROVED** [review]
This is ok to test on the server now.

File: project_scrum/wizard/project_scrum_test_task_view.xml [hunk 1]
```diff
@@ -10,11 +10,13 @@
 <!--        </record>-->
         
         <record model="ir.actions.server" id="ps_action_server_creat_test_task">
-            <field name="name">Creat Tasks from Test Cases</field>
+            <field name="name">Create Tasks from Test Cases</field>
             <field name="model_id" ref="model_project_scrum_sprint"/>
             <field name="state">code</field>
             <field name="type">ir.actions.server</field>
+<!--            <field name="view_mode">form</field>-->
```
Comment (daerio) [inline project_scrum/wizard/project_scrum_test_task_view.xml]: delete

File: project_scrum/wizard/project_scrum_test_task_view.xml [hunk 2]
```diff
@@ -10,11 +10,13 @@
...
             <field name="code">model.test_task()</field>
+            <field name="binding_model_id" ref="project_scrum.model_project_scrum_sprint" />
 <!--            <field name="menu_ir_values_id" ref="ps_make_test_task"/>-->
```
Comment (daerio) [inline project_scrum/wizard/project_scrum_test_task_view.xml]: delete

File: project_scrum/models/project_scrum.py [hunk 3]
```diff
@@ -199,8 +202,10 @@ class project_user_stories(models.Model):
     test_count = fields.Integer(compute = '_test_count', store=True)
     sequence = fields.Integer('Sequence')
     company_id = fields.Many2one(related='project_id.company_id')
-    #has_task = fields.Boolean()
-    #has_test = fields.Boolean()
+
```
Comment (daerio) [inline project_scrum/models/project_scrum.py:205]: delete

File: project_scrum/models/project_scrum.py [hunk 4]
```diff
@@ -601,6 +606,10 @@ class test_case(models.Model):
     stats_test = fields.Selection([('draft','Draft'),('in progress','In Progress'),('cancel','Cancelled')], string='State', required=False)
     company_id = fields.Many2one(related='project_id.company_id')
```
Comment (daerio) [inline project_scrum/models/project_scrum.py:608]: delete

**Review by daerio — CHANGES_REQUESTED** [review]
can you clean up these lines please?

**Review by daerio — APPROVED** [review]

**Pull request merged**